src/nvim/api/private/helpers.c

```diff
@@ -687,6 +703,22 @@ String cstr_to_string(const char *str)
     };
 }
 
+/// Copies buffer to an allocated String.
+/// The resulting string is also NUL-terminated, to facilitate interoperating
+/// with code using C strings.
+///
+/// @param str the buffer to copy
+/// @return the resulting String, if the input string was NULL, an
+///         empty String is returned
+String buf_to_string(const char *buf, size_t len)
```

Comment:
maybe name it `cbuf_to_string` in analogy with the others.

Reply:
`str` and `char` both refers to "c types as opposed to their API representation". But the alliteration is nice I suppose....
Yes, seems `size` is more common.